LEAF_Request_Portal/sources/Shortener.php

```diff
@@ -54,7 +54,7 @@ private function decodeShortUID($in)
         return $res - $this->offset;
     }
 
-    public function getFormQuery($shortUID) {
+    public function getFormQuery($shortUID, $form) {
```

Comment:
Wait, why is form here?

Reply:
Because before Form was instantiated in the method. for various reasons I couldn't do that anymore so now I'm passing it in so it can be used. This was done in several places in this step. It's a start to getting instantiation of classes out of other classes. If a class is needed in a different class it really should be passed in not instantiated on the fly.